pkg/api/server.go

```diff
@@ -119,7 +120,8 @@ func (s *Server) registerHandlers() {
 	s.router.HandleFunc("/token", s.tokenGenerateHandler).Methods("POST")
 	s.router.HandleFunc("/token/validate", s.tokenValidateHandler).Methods("GET")
 	s.router.HandleFunc("/api/info", s.infoHandler).Methods("GET")
-	s.router.HandleFunc("/api/echo", s.echoHandler).Methods("POST")
+	s.router.HandleFunc("/api/echo", s.echoHandler).Methods("POST,PUT")
+	s.router.HandleFunc("/api/echo{echo:.*}", s.echoHandler).Methods("POST,PUT")
```

Comment:
This should be consistent with the path patterns used above (currently "api/echo{echo:.*}" vs "/echo/{echo:.*}")

Reply:
This is consistent with the previous route paths; both `/api/echo` and `/echo` were exposed previously.